### DISCUSSION TITLE
Get unnormal spliced/unspliced proportion

### DISCUSSION THREAD

**Discussion by HuangDDU** [theislab/scvelo#896] [Q&A] [answered]
...
I use scvelo on my data and get unnormal spliced/unspliced proportion. Can I continue scVelo proportion on my data. 

```python
# paste your code here, if applicable
scv.pl.proportions(adata,save="proportion.pdf")
```

<details>
![scvelo_proportions_proportion](https://user-images.githubusercontent.com/41979954/176337588-aa7c4e49-d254-4d86-b294-13f2616761f4.png)
<summary>The proportion of spliced and unspliced mRNA is 2:8, which  is just oppsite to the wanted condition of 8:2 </summary>

```pytb
# paste the error output here, if applicable
The proportion of spliced and unspliced mRNA is 2:8, which  is just oppsite to the wanted condition of 8:2.
```
</details>

<details> <summary> Versions </summary>

```pytb
# paste the ouput of scv.logging.print_versions() here
scvelo==0.2.4  scanpy==1.9.1  anndata==0.8.0  loompy==3.0.7  numpy==1.21.5  scipy==1.8.0  matplotlib==3.5.2  sklearn==1.0.2  pandas==1.4.2 
```
</details>

**Comment by WeilerP** [comment, marked as answer]
@HuangDDU, yes, in theory, you can continue. I'd be cautious, though. A Couple of thoughts:
1. `scv.pl.proportions(adata)` is intended to be used with raw counts. Otherwise, this results in unexpected behavior.
2. To understand if RNA velocity can be inferred for your data, you need to have a look at the phase portraits. See e.g. #216 or #462.
